src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8919,7 +8929,7 @@ Real64 CalcEmmelRoof(EnergyPlusData &state,
                                       "CalcEmmelRoof: Convection model wind angle calculation suspect and high theta correlation",
                                       state.dataConvectionCoefficient->CalcEmmelRoofErrorIDX);
 
-        Hf = 3.54 * std::pow(WindAt10m, 0.76);
+        Hf = 3.67 * std::pow(WindAt10m, 0.85);
```

Comment:
I'm not sure it can even make it here, TBH. I updated this to pull from the roof correlation, but this probably needs to be double-checked (by me) and removed if it's inaccessible.

Reply:
I updated the error handling in these functions and fixed the theta angle calculation so that it's always bounded between 0-180.